File: backend_parcial/models/Member.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const MemberSchema = new Schema({
  name: { type: String, required: true },
  planId: { type: Schema.Types.ObjectId, ref: 'Plan', required: true },
  contributionPerMonth: { type: Number, required: true },
  joinedAt: { type: Date, default: Date.now }
});

module.exports = mongoose.model('Member', MemberSchema);

File: app/src/main/java/com/example/plan_de_ahorro/data/model/Member.kt
package com.example.plan_de_ahorro.data.model

import com.google.gson.annotations.SerializedName

data class Member(
    @SerializedName("_id") val id: String? = null,
    val name: String,
    val planId: String,
    val contributionPerMonth: Double,
    val joinedAt: String? = null
)
